### PR TITLE
Added Popover functionality to the list of authors.

### LINKED ISSUE #6923: Add popover to list authors on wiki page headings
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### What is to be done.
[Wiki pages](https://publiclab.org/wiki/onecranston) now have this great header showing Authors, Revisions, and the talk page:

image

Let's make it so you can click the Authors item and see a [Bootstrap popover](https://getbootstrap.com/docs/4.3/components/popovers/) listing the authors' usernames.

The code for that is here:

https://github.com/publiclab/plots2/blob/6d417a384801e4c20b77c4d6360788f19a78b41c/app/views/wiki/_header.html.erb#L34-L36

We can get the authors listing using `node.authors`, as shown here:

https://github.com/publiclab/plots2/blob/6d417a384801e4c20b77c4d6360788f19a78b41c/app/models/node.rb#L278-L280

And for each one, we could do the following so that their names are clickable and lead to their profile pages!

```html
<ul>
  <% node.authors.each do |author| %>
  <li><a href="/profile/<%= author.username %>">@<%= author.username %></a></li>
  <% end %>
</ul>
```
That code can go inside the popover!

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the file [$FILENAME]($BRANCH_URL) in the `$REPO` repository (press the little pen Icon) and edit the line as shown below. 

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
$DIFF
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it. 

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

### PR DESCRIPTION
**Pull request by Priyaraj17**
Fixes #6923 (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

Fixes_Issue link: https://github.com/publiclab/plots2/issues/6923

Here is a screenshot:
image

### CODE REVIEW

**Comment by Priyaraj17** [comment]
I have made a pull request, If it doesn't work, I will try to add more functionalities to it.

**Comment by cesswairimu** [comment]
thanks so much for opening this @Priyaraj17  :tada: could you please add the issue link to the issue this pull request fixes on your pull request on this line "Fixes issue_link" this will help other contributors while reviewing your changes. It also helps in automatically closing the issue when the pull request is merged. Thanks

File: app/views/dashboard/_node_meta.html.erb [hunk 1]
```diff
@@ -25,7 +25,11 @@
 </div>
 
 <div class="card-meta-links">
+<<<<<<< HEAD
```
Comment (govindgoel) [inline app/views/dashboard/_node_meta.html.erb:28]: need to remove this.
Same in lines 30,32.

**Comment by Priyaraj17** [comment]
> thanks so much for opening this @Priyaraj17  could you please add the issue link to the issue this pull request fixes on your pull request on this line "Fixes issue_link" this will help other contributors while reviewing your changes. It also helps in automatically closing the issue when the pull request is merged. Thanks

Sir, I have added the link now.

File: app/views/dashboard/_node_meta.html.erb (hunk 1 above)
Comment (Priyaraj17) [reply app/views/dashboard/_node_meta.html.erb:28]: Should I make another PR?

Comment (govindgoel) [reply app/views/dashboard/_node_meta.html.erb:28]: Just push another commit reflecting these changes.

Comment (Priyaraj17) [reply app/views/dashboard/_node_meta.html.erb:28]: Sorry, I didn't see your message. I will change this now.

Comment (Priyaraj17) [reply app/views/dashboard/_node_meta.html.erb]: Sir, I have updated the file.

File: app/views/wiki/_header.html.erb [hunk 2]
```diff
@@ -62,3 +68,5 @@
   </ul>
 </nav>
 <% end %>
+
```
Comment (govindgoel) [inline app/views/wiki/_header.html.erb:71]: can you check this there are two lines added in the EOF.

**Comment by Priyaraj17** [comment]
@govindgoel sir should I work on other issues?

File: app/views/wiki/_header.html.erb [hunk 3]
```diff
@@ -31,7 +31,13 @@
 
     <div style="padding-top:8px;text-align:center">
       <span class="d-none d-lg-inline">
-        <span rel="tooltip" title="The number of authors for this page." data-placement="top"><i style="color:#888;" class="fa fa-user"></i> <%= number_with_delimiter(@node.authors.size) %></span>  |
+        <span rel="popover" style='cursor: pointer;' title="Our Authors" data-html="true" data-placement="top" data-content= "
```
Comment (cesswairimu) [inline app/views/wiki/_header.html.erb:34]: Hi @Priyaraj17, sorry I didn't see the issue link on the bottom before...maybe we could edit the title to just `Authors` what do you think? Thanks

**Review by cesswairimu — CHANGES_REQUESTED** [review]

**Comment by cesswairimu** [comment]
@Priyaraj17 and yes you can work on other issues too. Thanks

File: app/views/wiki/_header.html.erb (hunk 3 above)
Comment (Priyaraj17) [reply app/views/wiki/_header.html.erb:34]: Yeah sure.

Comment (Priyaraj17) [reply app/views/wiki/_header.html.erb]: I have made the changes as requested.

**Comment by Priyaraj17** [comment]
> @Priyaraj17 and yes you can work on other issues too. Thanks

Sure. I will look for them

**Review by cesswairimu — APPROVED** [review]
Fantastic :tada: , thanks again!!

**Pull request merged**